shared/src/main/java/com/faforever/neroxis/map/SCMap.java

```diff
@@ -80,6 +80,7 @@ public class SCMap {
     private String filePrefix = "";
     private String script = "";
     private String skyCubePath = "/textures/environment/defaultskycube.dds";
+    @Setter
```

Comment:
From my earlier comment and thinking about it more I think it is still preferable that the map biome be immutable. So this means that the various sub style generators need to be chosen in the StyleGenerator initialize function so the biome can be loaded and the map created with it in that function. Then the prop generators reading the biome from the map should be fine since it never changes.